src/acquisition/covidcast/direction.py

```diff
@@ -0,0 +1,47 @@
+"""Classifies the direction of a signal based on its recent trajectory."""
+
+# third party
+import numpy
```

Comment:
if numpy isn't used anywhere else, consider replacing use of `numpy.abs` with python base `abs` and `numpy.sign` with a conditional

Reply:
Peace of mind about the implementation of `numpy.sign` might be worth it though.

Reply:
i agree it is preferable to use built-in `abs`. done.

however, what would be a benefit to reimplementing `sign`? if it's strictly a matter of shedding a dependency, note that scipy itself depends on numpy. this is confirmed by the timing report in the unit test. in particular, the incremental cost of importing numpy after scipy is negligible.

```
    4 import numpy                                                                     1x         163 ms     163 ms    
    5 import scipy.stats                                                               1x         270 ms     270 ms    
```

```
    4 import scipy.stats                                                               1x         437 ms     437 ms    
    5 import numpy                                                                     1x                              
```